src/test/php/web/unittest/server/SAPITest.class.php

```diff
@@ -214,4 +214,12 @@ function($part) { return nameof($part); },
       $this->parts($this->upload('test.txt', 'text/plain'))
     ));
   }
+
+  #[Test]
+  public function parameter_unnecessary_urlencode_regression() {
+    $_REQUEST= ['varname' => 'the value'];
+    $fixture= new SAPI();
+    $parts = iterator_to_array($fixture->parts(''));
+    $this->assertNotEquals('the+value', $parts['varname']->value());
```

Comment:
Why don't you simply check for `assertEquals('the value', $parts...)`? This test of yours would succeed even if suddenly `$parts['varname']->value()` returned *NULL*.

Reply:
Yea the test would be  more versatile if is used assertEquel. The thjinking behind the notEqual was to just cover the regression.